test.js

```diff
@@ -1,7 +1,7 @@
 /* eslint-disable no-extend-native */
 Array.prototype.findIndex = undefined;
 import test from 'ava';
-import fn from './';
+import fn from '.';
```

Comment:
I'm keeping this for consistency. https://github.com/sindresorhus/eslint-plugin-xo/issues/9